collections/scheduledMessages.js

```diff
@@ -1,6 +1,8 @@
 ScheduledMessages = new Mongo.Collection('scheduledMessages')
 ScheduledMessages.attachSchema(new SimpleSchema({
   _id: {type: String, regEx: SimpleSchema.RegEx.Id, optional: true},
+  name: {type: String, optional: true},
```

Comment:
Should we maybe add a default of empty string here just to ensure that the schema is predictable?

Reply:
Yeah that's a good idea. I'll make that change.

Reply:
Actually I'm just going to switch back to making them optional because it's unclear whether SimpleSchema would allow you to save a document with an empty string as a value: https://forums.meteor.com/t/simpleschema-add-empty-string-to-mongodb/16831/2

Reply:
NVM. I think this suggests that the empty string will be saved regardless: 
https://github.com/aldeed/simple-schema-js#defaultvalue
```
Note the following points of confusion:

A default value itself is not cleaned. So, for example, if your default value is "", it will not be removed by the removeEmptyStrings operation in the cleaning.
A default value is added only if there isn't a value set AND the parent object exists. Usually this is what you want, but if you need to ensure that it will always be added, you can add defaultValue: {} to all ancestor objects.
```